transcom/mymove
# B-20531-INT code coverage

## Summary
Code coverage passed on INT for 20531, but failed on main. This is an unrelated test update to increase coverage, so 20531 to main will pass coverage failure

TO TEST
Make sure we are all green checks, or if you're feeling extra, pull down the branch and run `make client_test`
